### PR TITLE
Add ART SRO bundle tests

### PR DESCRIPTION
**Pull request by pacevedom**
Add ART bundles to PSAP SRO ci-artifacts tests.

### CODE REVIEW

**Comment by kpouget** [comment]
@pacevedom you need to run `make ci-operator-config jobs` before pushing modifications to this repo, it will check the consistency (stanza order, unknown stanza will be removed, etc) and generate `job` files

File: ci-operator/jobs/openshift-psap/ci-artifacts/openshift-psap-ci-artifacts-master-presubmits.yaml [hunk 1]
```diff
@@ -1015,6 +1015,72 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )sro-e2e-master,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```
Comment (kpouget) [inline ci-operator/jobs/openshift-psap/ci-artifacts/openshift-psap-ci-artifacts-master-presubmits.yaml:1019]: can you manually edit this file and turn off `always_run`?
with clusters being resource intensive, we (I) think it's better to keep the testing "human controlled" ;-)

the following `make jobs` will take care not to override this setting

**Review by kpouget — COMMENTED** [review]
seems good @pacevedom, the test failed the expected way!

I guess it's on purpose that you only added a PR (presubmit) test and no periodic testing, right?

only need to turn of the `always_run` flag and that will be good for merging

**Comment by pacevedom** [comment]
@kpouget no periodics for now until I verify everything is working as expected. Thats also why this is marked as WIP.

**Comment by kpouget** [comment]
@pacevedom do you want to merge this PR, to enable `/test sro-e2e-master-art` on [your] PR?

**Comment by pacevedom** [comment]
@kpouget sure, thanks!

**Comment by kpouget** [comment]
/lgtm

there you go, remove the `WIP` flag and in a few minutes the `/test` will be available in the PR!

**Pull request merged**